src/components/Map/Map.js

```diff
@@ -61,7 +61,10 @@ class Map extends Component {
 		// create the base map
 		const map = new LeafletMap(`map-container-${mapId}`, {
 			scrollWheelZoom: false,
-			zoomControl: showZoomControls
+			zoomControl: showZoomControls,
+			// control dragging and tap in order to prevent scroll hijacking
```

Comment:
Can we add these options after instantiation? Docs didn't seem to have a clear answer. 

I noticed Leaflet class contains a Browser utility that can detect mobile devices. Wondering can conditionally passing in the zoom/dragging options using the Browser utility is a better option?

Reply:
ah nice find! I think that the consuming application should be in control of what's "mobile" or not (and whether to toggle), so this passthrough should be fine for now